server/database.js

```diff
@@ -1,28 +1,90 @@
 const mysql = require('mysql');
+require('dotenv').config();
 
 const loginInfo = {
-    host: 'mycoolsql',
-    port: '3306',
-    user: 'root',
-    password: 'sheep',
-    database: 'featurefilm'
+  host: 'mycoolsql',
+  port: '3306',
+  user: 'root',
+  password: 'MattM',
+  database: 'featurefilm',
 };
 
 const connection = mysql.createConnection(loginInfo);
 
+// create
+const createFeaturedFilm = (params) => {
+  const createQuery = `INSERT INTO Features (
```

Comment:
I think this is a good opportunity to make use of the SET query, you could then pass "params" as the second argument to your query. It would be easier to read than ```VALUES (?,?,?,? ...)``` 

Totally up to you though, as long as the functionality's there.